src/login/cert_helpers.h

```diff
@@ -37,8 +37,8 @@ namespace certificateHelpers
                 {
                     ShowInfo(fmt::format("Found existing login.key"));
                 }
+                fclose(fileHandle);
```

Comment:
Use `utils::openFile` to open a file and get back a `unique_ptr` that'll close itself when it falls out of scope